src/styles/about.module.css

```diff
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css?family=Lilita+One|Quicksand:300&display=swap');
+.container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    position: relative;
+    margin: 3% 0;
+}
+
+.innerContainer {
+    width: 49%;
+    height: 430px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.font {
+    position: center;
+    font-family: 'Quicksand', sans-serif;
```

Comment:
You should probably define your font in the global.css

Reply:
I tried to put the font in global.css. But it doesn't work.